docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+
+Please proceed to create a notification workflow.
+
+1. Click “Workflows” on the left sidebar of your Novu dashboard.
+2. Click the “Create Workflow” button on the top right.
+3. The name of a new workflow is currently "Untitled." Rename it to a more suitable title.
+4. Select "Email" as the channel you want to add.
+   ![set-email.png] (https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/set-email_wavtrn.png)
+
+5. Click on the recently added channel, fill the email subject and click “Update”.
+   ![update_email_template.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/update_email_template_ivn0jv.png)
+
+6. Click on the “Test” tab and send a test email to verify your notification workflow.
+   ![send_test_email.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/send_test_email_ngzmth.png)
+
+You should get an email within seconds. Yaaay, you have successfully sent your first notification via the Novu dashboard! Now, let’s take it a step further to trigger notifications via code.
+
+## Create A Subscriber
+
+The recipients of a triggered notification are called subscribers.
+
+Click “Subscribers” on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign-up.
+
+![subscriber_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1688331839/Screenshot_2023-07-03_at_0.02.53_jmkhi3.png)
+
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+
+```python
+# Create a subscriber
+
+from novu.api.subscriber import SubscriberApi
+from novu.dto.subscriber import SubscriberDto
+   
+your_subscriber_id = "123"; # Replace this with a unique user ID that matches your database.
+
+subscriber = SubscriberApi("https://api.novu.co", "b23c55928da9e76702b74921a7fffee2")
+subscriber.create(SubscriberDto(
+        subscriber_id=your_subscriber_id,
+        email="abc@gmail.com",
+        first_name="John",
+        last_name="Doe"
+    )
+)
+```
+
+Run the code in your terminal like so:
+
+```bash
+python main.py # replace main.py with your file name
+```
+
+You should see the subscriber on your Novu dashboard.
+
+## Update A Subscriber
+
+To update the Subscriber details you can follow you can call the `put` method from `SubcriberApi`. Here is an example:
+
+```python
+# Update subscriber detail
+
+from novu.api.subscriber import SubscriberApi
+from novu.dto.subscriber import SubscriberDto
+subscriber = SubscriberApi("https://api.novu.co", "<YOUR_NOVU_API_KEY>")
+subscriber.put(subscriber=SubscriberDto(
+        subscriber_id="<You want to update>",
+        email="abc@gmail.com",
+        first_name="John",
+        last_name="Doe"
+))
+```
+
+Other valid fields that can be updated are `email`, `first_name` etc.
+
+<aside>
+Note: To make all of your app users subscribers, you need to programmatically add them to Novu.
```

Comment:
Note: To send notifications to all your users, you'll need to make them subscribers in Novu, which you can do by programmatically adding them to Novu.